schemas/JSON/settings/settings.schema.0.2.json

```diff
@@ -184,6 +184,15 @@
           "type": "integer",
           "default": 3,
           "minimum": 1
+        },
+        "archiveExtractionMethod": {
+          "description": "Controls the use of tar.exe for extracting packages.",
```

Comment:
nit: update the description. It's for specifying archive extraction method, not just tar.exe.